Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+import json
+import collections
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+API_KEY = ""
+INSECURE = demisto.params().get('insecure')
+BASE_URL = 'https://api.github.com/'
+
+'''HELPER FUNCTIONS'''
+
+
+def http_request(method, URL_SUFFIX, json=None):
+    if method is 'GET':
+        headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + API_KEY
+        }
+    elif method is 'POST':
+        if not API_KEY:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json'
+            }
+        else:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+                'Authorization': 'Bearer ' + API_KEY
+            }
+    r = requests.request(
+        method,
+        BASE_URL + URL_SUFFIX,
+        data=json,
+        headers=headers,
+        verify=INSECURE
+    )
+    if r.status_code not in [200, 201]:
+        return_error('Error in API call [%d] - %s' % (r.status_code, r.reason))
+    return r.json()
+
+
+def filter_issues_by(response):
+    issues = {}
+    for issue in response:
+        issues[issue['number']] = issue['title']
+    return issues
+
+
+# Allows nested keys to be accesible
+def makehash():
+    return collections.defaultdict(makehash)
+
+
+'''MAIN FUNCTIONS'''
+
+
+def list_all_issues():
+    r = http_request('GET', 'issues')
+    results = []
+    for issue in r:
+        issue_details = makehash()
+
+        issue_details['number'] = issue['number']
+        issue_details['ID'] = str(issue['id'])
+        issue_details['title'] = issue['title']
+        issue_details['link'] = issue['url']
+        issue_details['state'] = issue['state']
+
+        results.append(issue_details)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': r,
+        'HumanReadable': tableToMarkdown('Issues list:', results, ['number', 'ID', 'title', 'state', 'link']),
+        'EntryContext': {
+            'GitHub.Issues(val.ID==obj.ID)': results
+        }
+    })
+    return r
+
+
+def create_issue(title, body, assignees, labels):
+    query = {'title': title, 'body': body, 'assignees': assignees, 'labels': labels}
+    search = json.dumps(query)
```

Comment:
use more meaningful names, in case we will come to this code in one year we should be able to read it and don't do any deductions our own
```suggestion
    search_query = json.dumps(query)
```